test/maps/lua_testsuite.wmf/scripting/militarysite.lua

```diff
@@ -60,6 +62,13 @@ function militarysite_tests:test_set_soldiers_add_and_remove()
    assert_equal(0, self.fortress:get_soldiers({0,0,0,0}))
    assert_equal(0, self.fortress:get_soldiers({3,0,0,1}))
    assert_equal(1, self.fortress:get_soldiers({3,2,0,1}))
+   -- correct type of bob.descr (inclusive attribute)
+   local soldier_bobs = self.fortress.fields[1].bobs
+   assert_equal(1, #soldier_bobs)
```

Comment:
This line fails 3 times in one testsuite.
Once it returns 695 ([L951](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959842054#step:7:951))
and twice 115 ([L3405](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959842054#step:7:3405) and [L3782](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959842054#step:7:3782))
Will not be easy to find one specific soldier among all the bobs. :confused: 

And [test_road_crosses_another](https://github.com/widelands/widelands/actions/runs/5469791867/jobs/9959843036#step:7:3526) fails !???  This happened as well in [pr 5993](https://github.com/widelands/widelands/pull/5993#pullrequestreview-1514327735) but no more when there was no other failure.

Reply:
All bobs which were in the former buildings are standing on this spot.
I move the building spot to a different place.
(As we do not play, the unnecessary 600 bobs do probably not matter and not use up too much memory/cpu.)

Reply:
Some of the fortresses could [not be created](https://github.com/widelands/widelands/actions/runs/5474140062/jobs/9969469918#step:7:910)
So I moved them to a area which is very likely free and inside the conquered area. Then the tests passed.